backend/services/mailer.js

```diff
@@ -39,8 +39,69 @@ function sendEmail(email) {
   return true;
 }
 
+function sendRegistrationConfirmationEmail(baseURL, receiverEmail, receiverName, receiverConfirmationCode) {
+  const msg = {
+    intention:"Invitation",
+    to: receiverEmail,
+    from: {
+      email: "noreply.evoter@gmail.com",
+      name: "eVoter Support",
+    },
+    subject: `eVoter - Registration Confirmation for ${receiverName}`,
+    text: "using html", // using HTML instead of text, but text cannot be empty
+    html: `<h1>eVoter - Registration Confirmation for ${receiverName}</h1>
+            <h2>Hello ${receiverName},</h2>
+            <p>Thank you for registration. Please confirm your email by clicking on the following link:</p>
+            <a href=http://${baseURL}/users/confirm/${receiverConfirmationCode}> Click here</a>
+            </div>`,
+  };
+  console.log(`http://${baseURL}/users/confirm/${receiverConfirmationCode}`);
+  return sendEmail(msg);
+}
+
+function sendRegistrationInvitationEmail(receiverEmail) {
+  const msg = {
+    intention:"Invitation",
+    to: receiverEmail,
+    from: {
+      email: "noreply.evoter@gmail.com",
+      name: "eVoter Support",
+    },
+    subject: `eVoter - Invitation to Sign Up eVoter`,
+    text: "using html", // using HTML instead of text, but text cannot be empty
+    html: `<h1>eVoter - Invitation to Sign Up eVoter</h1>
+            <h2>Hello,</h2>
+            <p>You are invited to a poll on eVoter. Please sign up an account:</p>
```

Comment:
Since we aren't assigning a voter to a poll if we send them a registration invitation email, we can probably change the message to something like "You have been invited to sign up for an account in eVoter. Please sign up now!" or similar